src/commands/ml-models-list.ts

```diff
@@ -1,12 +1,49 @@
+import * as clc from "cli-color";
 import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
 import { FirebaseModel } from "../ml/models";
+import { requirePermissions } from "../requirePermissions";
+import { ensureFirebaseMlApiEnabled, getTableForModelList } from "../ml/mlHelper";
+import * as mlApi from "../ml/mlApi";
+import * as logger from "../logger";
+import * as utils from "../utils";
+import { logPrefix } from "../ml/mlHelper";
 
 export default new Command("ml:models:list")
-  .description("lists all the Firebase ML Models in the current project.")
+  .description("list all the Firebase ML Models in the current project.")
+  .help("TODO")
```

Comment:
Leaving this comment here as a DO NOT SUBMIT for this TODO

Reply:
This isn't going into master, it's going into feature-ml. When I'm done with all the code, Kevin will do all the help text in a separate PR before we merge the whole feature into master. (I already talked to him about this and he agreed).